webdriver/tests/bidi/browsing_context/set_viewport/set_viewport.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from ... import get_viewport_dimensions
+
+@pytest.mark.asyncio
```

Comment:
```suggestion

@pytest.mark.asyncio
```

Again please use two empty lines as separator for all the top-level methods including here.